common/data_refinery_common/rna_seq.py

```diff
@@ -36,17 +37,15 @@ def should_run_tximport(experiment: Experiment, results, is_tximport_job: bool):
 
     results is a queryset of ComputationalResults for the samples that had salmon quant run on them.
     """
-    num_quantified = results.count()
+    num_quantified = len(results)
```

Comment:
Well the primary reason for switching here wasn't because of an optimization between count/len, it was because it's not a quesryset any more. I just read the docstring right above this that is out of date now. I'll update that.